azure-pipelines/build/linux/du/templates/do-docker-jobs.yml

```diff
@@ -17,6 +17,8 @@ jobs:
       targetOsArch: ${{parameters.targetOsArch}}
       imageVersion: ${{parameters.imageVersion}}
       config: debug
+  - publish: $(Build.ArtifactStagingDirectory)
+    artifact: 'doclient-drop-${{parameters.targetOsArch}}-debug'
```

Comment:
>drop [](http://example.com/codeflow?start=24&length=4)

is this naming convention new?

Reply:
Yes. The 1ES tasks also publish artifacts/drop, so I'm making ours distinguishable.

Reply:
I see. What is different from the artifacts that the 1ES task produces and ours?

Reply:
Example artifacts: [https://dev.azure.com/deliveryoptimization/client/_build/results?buildId=4469&view=artifacts&pathAsName=false&type=publishedArtifacts](https://dev.azure.com/deliveryoptimization/client/_build/results?buildId=4469&view=artifacts&pathAsName=false&type=publishedArtifacts)

Reply:
thank you, that helps. The doclient convention makes sense; wasn't too sure about the "drop", but I guess its consistent.